tests/flows/test_simple_flow.py

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.models_structures import ErcCreateData
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def test_simple_flow():
+
+    private_key = os.getenv("TEST_PRIVATE_KEY1")
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+
+    wallet = Wallet(
+        ocean.web3, private_key, config.block_confirmations, config.transaction_timeout
+    )
+
+    erc721_token = ocean.create_nft_token(
```

Comment:
Can we have some assertions on erc721 and erc20 tokens instead of prints?

Reply:
sure, give me a minute :)